examples/scf_properties/batch/sdk/run.py

```diff
@@ -0,0 +1,132 @@
+import os
+import pathlib
+
+from promethium_sdk.utils import base64encode
+from promethium_sdk.client import PromethiumClient
+from promethium_sdk.models import (
+    CreateSinglePointCalculationWorkflowRequest,
```

Comment:
We should add a comment here and in the other example that this needs to be run with SDK version >= 0.3.12. (From my testing it will run on prior versions, but the `scf_properties` will not be added to the `payload` object)

Reply:
Ah, thanks.  I also added a note to the README file.